backend/src/main/java/heartbeat/service/board/jira/JiraService.java

```diff
@@ -370,10 +371,11 @@ private JiraCardWithFields getAllCards(BoardType boardType, URI baseUrl, BoardRe
 		else {
 			throw new BadRequestException("boardType param is not correct");
 		}
-		return getCardList(baseUrl, boardRequestParam, jql, "all");
+		return getCardList(baseUrl, boardRequestParam, jql, "all", null);
 	}
 
-	private AllCardsResponseDTO formatAllCards(String allCardResponse, List<TargetField> targetFields) {
+	private AllCardsResponseDTO formatAllCards(String allCardResponse, List<TargetField> targetFields,
```

Comment:
The method 'formatAllCards(String, List, List)' has an NPath complexity of 616, current threshold is 200